app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -567,15 +561,35 @@ class SettingsFragment : PreferenceFragmentCompat() {
     internal fun setupHttpsOnlyPreferences() {
         val httpsOnlyPreference =
             requirePreference<Preference>(R.string.pref_key_https_only_settings)
-        httpsOnlyPreference.summary = context?.let {
-            if (it.settings().shouldUseHttpsOnly) {
+        httpsOnlyPreference.summary = context?.settings()?.let {
+            if (it.shouldUseHttpsOnlyInPrivateTabsOnly) {
+                getString(R.string.preferences_https_only_in_private_tabs)
```

Comment:
This would get
> Enable only in private tabs

I would argue
> Enable**d** only in private tabs 
would be better suited to express an already set behavior, not the description of a future one.

Also, if we are to offer more context and use `Enabled only in private tab` as a description of HTTPs used only in private mode we could also use `Enabled in all tabs` when HTTPs is enabled as such and maybe even `Disabled` when HTTPs-only mode is turned off.